src/api/cancel.ts

```diff
@@ -0,0 +1,15 @@
+import { IncomingRequestMessage } from "../core";
+
+/**
+ * A request to end a {@link Session} (incoming CANCEL).
+ * @public
+ */
+export class Cancel {
+  /** @internal */
+  public constructor(private incomingCancelRequest: IncomingRequestMessage) {}
+
+  /** Incoming ACK request message. */
```

Comment:
/** Incoming ACK request message. */
Why ACK?

Reply:
Yep, it's a copy-paste from another object, I will fix.